modules/lib/lib-node/src/main/resources/lib/xp/node.ts

```diff
@@ -438,12 +438,12 @@ interface FindChildrenHandlerParams {
     setRecursive(recursive: boolean): void;
 }
 
-export interface DuplicateParams {
+export interface DuplicateParams<NodeData = Record<string, unknown>> {
```

Comment:
If this should be consistent with the way it's solved in `CreateNodeParams` and `ModifyNodeParams` the default value should just be `unknown`.

```suggestion
export interface DuplicateParams<NodeData = unknown> {
```

Reply:
Since we don't actually support anything but objects that uses string as the key, would this be better?
```suggestion
export interface DuplicateParams<NodeData extends Record<string, unknown> = Record<string, unknown>> {
```

Reply:
And to the same for `CreateNodeParams` and `ModifyNodeParams`?

Reply:
I don't think using `extends Record<string, unknown>` will work. I tried it out, and it rejects my (legal) `interface` with the  shape that I want.

Reply:
Regarding my initial suggestion, I don't feel strongly about this. It's was more a comment to try to maintain consistency of code across functions.

Either way is good. 

Developers will most likely not interact with this shape independently, but only trough using the `duplicate()`-function which is the one that actually takes the type parameter, and has a default type parameter.